### ISSUE TITLE
Support CSS variables in [ngStyle] and [style.--foo] bindings

### ISSUE THREAD

**Issue by leonsenft** [dart-lang/angular#501]

**Comment by leonsenft** [comment]
Blocked by dart-lang/sdk#26819.

**Comment by matanlurey** [comment]
It is blocked? Or is it just not even something we need to change at all ourselves?

**Comment by matanlurey** [comment]
i.e. seems like a duplicate of that issue versus it's own issue

**Comment by leonsenft** [comment]
Yep! You're right, I don't think there is actually an issue on our end. Fixing the dart:html error should fix this.
